### PR TITLE
Translate docs/book/zh/src/kubernetes/cri.md to zh

### PR DESCRIPTION
**Pull request by zswaaa**

### CODE REVIEW

File: docs/book/zh/src/kubernetes/cri.md [hunk 1]
```diff
@@ -1 +1,7 @@
 # CRI runtimes
+
+高级容器运行时，例如[CRI-O](https://cri-o.io/)和[containerd](https://containerd.io/)，从注册表拉取容器镜像（例如，Docker Hub），在磁盘上管理这些镜像，并启动一个低级容器运行时来运行容器进程。
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/cri.md:3]: 英文前后空一格

File: docs/book/zh/src/kubernetes/cri.md [hunk 2]
```diff
@@ -1 +1,7 @@
...
+在本章节中，你可以看到一些针对CRI-O和containerd的具体教程。
```
Comment (alabulei1) [inline docs/book/zh/src/kubernetes/cri.md:4]: 英文前后空一格

**Review by alabulei1 — COMMENTED** [review]
@zswaaa  Thanks for your contribution. Please check out my feedback.

**Comment by zswaaa** [comment]
@alabulei1 Thanks for your review, all the problems pointed out above have been fixed.

**Review by hydai — APPROVED** [review]

**Pull request merged**